_data/communication.yml

```diff
@@ -180,6 +180,14 @@ websites:
       software: Yes
       doc: https://peerio.zendesk.com/hc/en-us/articles/202933829-How-do-I-setup-two-factor-authentication-
 
+    - name: Riot.im
+      url: https://about.riot.im/
+      img: riot.png
+      twitter: RiotChat
+      tfa: No
+      doc: https://github.com/vector-im/riot-web/issues/2772
```

Comment:
So I ran the PR locally and `doc` links are ignored when tfa is set to No.
![](https://tempfiles.carlgo11.com/download/D5CAFFA58D3FD7/?p=6oCrxfN7v)

Therefore, please remove the doc link:
```suggestion
```